serverless/develop/serverless-using-brokers.adoc

```diff
@@ -15,7 +15,7 @@ Events can be sent from an event source to a broker as an HTTP POST request.
 After events have entered the broker, they can be filtered by https://github.com/cloudevents/spec/blob/v1.0/spec.md#context-attributes[CloudEvent attributes] using triggers, and sent as an HTTP POST request to an event sink.
 
```

Comment:
```suggestion

Brokers have multiple implementations which have different event delivery guarantees and might use different technologies (e.g Apache Kafka). You can choose the broker implementation when creating a broker by providing a broker class otherwise the default broker class configured by cluster administrators will be used.

You can use the following broker implementations with {ServerlessProductName}

=== Channel-based broker

The Channel-based broker implementation internally uses channels for event delivery. The channel implementation will be determined when you create a broker based on the broker configuration.

The channel-based broker provides different event delivery guarantees based on the channel implementation a broker instance uses. For instance, a broker using the InMemoryChannel is useful for development and testing purposes but does not provide the event delivery guarantees for production environments, while a broker using the KafkaChannel provides the delivery guarantees needed for a production environment.

=== Kafka broker
:FeatureName: Kafka broker
include::modules/technology-preview.adoc[leveloffset=+1]

The Kafka broker is a broker implementation that uses Kafka internally to provide at least once delivery guarantees. It supports any Kafka version, and has a native integration with Kafka for storing and routing events.
```

Reply:
That's really good section here for clarification on the differences between the choices we now have. Thanks @devguyio 

+1 on using this section!

Reply:
AWESOME, thanks @devguyio 😄